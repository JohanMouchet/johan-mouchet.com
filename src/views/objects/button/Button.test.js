import React from "react";
import { shallow } from "enzyme";
import { Button, ButtonSocial, ButtonSocialBadge } from ".";

it("Renders <Button /> without crashing", () => {
  shallow(<Button />);
});

it("Renders <ButtonSocial /> without crashing", () => {
  shallow(<ButtonSocial />);
});

it("Renders <ButtonSocialBadge /> without crashing", () => {
  shallow(<ButtonSocialBadge />);
});
