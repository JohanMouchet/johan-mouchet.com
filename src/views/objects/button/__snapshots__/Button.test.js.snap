// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Button /> without crashing 1`] = `
<button
  className="o-button"
  type="button"
/>
`;

exports[`Renders <ButtonSocial /> without crashing 1`] = `
<a
  className="o-button o-button--social"
>
  <span
    className="o-button__icon"
  />
  <span
    className="o-button__text"
  />
</a>
`;

exports[`Renders <ButtonSocialBadge /> without crashing 1`] = `
<a
  className="o-button o-button--social o-button--social-badge"
  rel="noopener noreferrer"
  target="_blank"
>
  <span
    className="o-button__icon"
  />
</a>
`;
